RUFAS/task_manager.py

```diff
@@ -181,6 +186,70 @@ def start(
             export_input_data_to_csv=export_input_data_to_csv,
         )
 
+    def get_rufas_version(self) -> str:
+        """
+        Returns the current version of RUFAS.
+
+        Returns
+        -------
+        str
+            Version of RUFAS or "Unknown" if the person is using a Python version earlier than 3.11.
+        """
+        try:
+            with open(PYPROJECT_FILE_PATH, "rb") as pyproject_file:
```

Comment:
Does it have to be binary?
```suggestion
            with open(PYPROJECT_FILE_PATH, "r") as pyproject_file:
```

Reply:
It does. The tomllib load function expects a binary file (which is different from the third-party toml library that expects a normal text-mode file).